app/policies/consumer_role_policy.rb

```diff
@@ -18,6 +23,45 @@ def privacy?
     end
   end
 
+  # Determines if the current user has permission to upload ridp document.
+  # The user can upload the document if they are a primary family member,
+  # an active associated broker, or an admin in the individual market,
+  #
+  # @return [Boolean] Returns true if the user has permission to upload the document, false otherwise.
+  def ridp_document_upload?
+    return true if individual_market_non_ridp_primary_family_member?
+    return true if individual_market_admin?
+    return true if active_associated_individual_market_family_broker?
```

Comment:
@saipraveen18 Please also include access to the Broker Agency Staff Roles.
- `active_associated_individual_market_family_broker_staff?` or
- `active_associated_individual_market_ridp_verified_family_broker_staff?`

Reply:
added checks for `broker_agency_staff`